docs/docs/adapters/circom.md

```diff
@@ -0,0 +1,196 @@
+# Circom Adapter
+
+Mopro supports the integration of Circom circuits. For this, you need to have pre-built `zkey` and `wasm` files for your
+circuits. You can find more information on how to generate these files in
+the [Circom documentation](https://docs.circom.io).
+
+## Sample Project
+
+Explore how the Circom adapter is implemented by checking out this sample
+project: [mopro-app](https://github.com/vimwitch/mopro-app).
```

Comment:
can we use https://github.com/zkmopro/mopro/tree/main/test-e2e
this is where we maintain (and test) each adapter

Reply:
Do we then need a sample Halo2 app? As we have in this sample both Halo2 and Circom adapter?